net/src/net.cpp

```diff
@@ -405,6 +481,13 @@ int lua_serve(lua_State* L)
         }
         lua_pop(L, 1);
 
+        lua_getfield(L, 1, "allow_port_reuse");
```

Comment:
```suggestion
        lua_getfield(L, 1, "reuseport");
```
or `allowportreuse` to be luaucase